runtime/lua/vim/lsp.lua

```diff
@@ -672,13 +672,13 @@ function lsp.start_client(config)
   --- corresponding handler.
   ---
   ---@param method (string) LSP method name
-  ---@param params (table) The parameters for that method.
-  function dispatch.notification(method, params)
-    local _ = log.debug() and log.debug('notification', method, params)
+  ---@param result (table) The parameters for that method.
```

Comment:
is this docstring correct? result=parameters ?

Reply:
Changed this signature back